components/director/cmd/tenantfetcher-job/main.go

```diff
@@ -35,6 +35,7 @@ type config struct {
 	Features features.Config
 
 	TenantProvider       string        `envconfig:"APP_TENANT_PROVIDER"`
+	TenantsRegion        string        `envconfig:"default=central,APP_TENANTS_REGION"`
```

Comment:
Is `central` the desired value or we should override it per universe?

Reply:
We should override it, but I left one `central` here for the sake of open source :D I'll move the default value in the charts